wheel/build_wheel.sh

```diff
@@ -181,7 +181,7 @@ python setup.py bdist_wheel -d "$whl_tmp_dir"
 echo "Finished setup.py bdist_wheel at $(date)"
 
 echo "delocating wheel dependencies"
-retry pip install https://github.com/matthew-brett/delocate/archive/master.zip
+retry pip install https://github.com/matthew-brett/delocate/archive/a5ba7dcc8ccfc9aaa1411c63266bfaf87a4ff393.zip
```

Comment:
Use tags rather than hash sum
```suggestion
retry pip install https://github.com/matthew-brett/delocate/archive/refs/tags/0.10.2.zip
```